# compilation of bpf skeletons is not needed for the building of bpftool.
# also this causes some issues because it invokes a non arch specific clang
# which results in cros build errors.
diff --git a/Makefile b/Makefile
index b83bf5a..7aa969e 100644
--- a/Makefile
+++ b/Makefile
@@ -92,8 +92,7 @@ RM ?= rm -f
 
 FEATURE_USER = .bpftool
 
-FEATURE_TESTS := clang-bpf-co-re
-FEATURE_TESTS += llvm
+FEATURE_TESTS := llvm
 FEATURE_TESTS += libcap
 FEATURE_TESTS += libbfd
 FEATURE_TESTS += libbfd-liberty
